datasets/wikicorpus/wikicorpus.py

```diff
@@ -159,7 +159,7 @@ def _generate_examples(self, dirpath):
                             pass
                         elif row.startswith("ENDOFARTICLE") or row.startswith("\n"):
                             if len(words) > 1:  # some content besides only (. . Fp 0)
-                                yield id_, {
+                                yield (file_idx, row_idx), {
```

Comment:
Is there a reason to yield a tuple, differently from above, where it is yielded a string instead?
```suggestion
                                yield f"{file_idx}_{row_idx}", {
```

Reply:
Good catch ! This needs to be a string indeed - we don't support tuples anymore since we detect duplicate keys